src/Stratis.Bitcoin.IntegrationTests/BlockStoreTests.cs

```diff
@@ -148,9 +147,9 @@ public void BlockBroadcastInv()
         {
             using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var stratisNodeSync = builder.CreateStratisPowNode();
-                var stratisNode1 = builder.CreateStratisPowNode();
-                var stratisNode2 = builder.CreateStratisPowNode();
+                var stratisNodeSync = builder.CreateStratisBitcoinPowNode();
```

Comment:
hmm this name is problematic `CreateStratisBitcoinPowNode` we need to come up with a better way to distinguish the framework name and the token.

Reply:
I agree @dangershony ! I'll name the nodes with more intent.

Reply:
OK should be better now :)